tests/benchmarks/test_h2o.py

```diff
@@ -37,11 +37,13 @@
     }
 
 
-@pytest.fixture(params=list(DATASETS))
-def ddf(request, small_client):
-    if request.param not in enabled_datasets:
-        raise pytest.skip("Disabled by default config or H2O_DATASETS env variable")
+@pytest.mark.fixture(autouse=True)
+def client(small_client):
+    yield small_client
 
+
+@pytest.fixture(params=list(enabled_datasets), scope="module")
```

Comment:
This causes the order of the tests to become pseudo-random - which from memory is something that pytest has problems with.

Could you change both definitions of `enabled_datasets` above to be lists?

Reply:
I went with a sorted instead since `DATASETS` is a dict not a set

Reply:
> I don't mind changing it to lists but how is my version any different to what was here before?

previously it was iterating on dict keys, which retain insertion order